unit_test/fuzzing/spdm_unit_fuzzing_common/toolchain_harness.c

```diff
@@ -20,7 +20,15 @@
 size_t libspdm_alignment_size(size_t size)
 {
     size_t alignment;
+    size_t max_buffer_size;
+
     alignment = LIBSPDM_TEST_ALIGNMENT;
+    max_buffer_size = libspdm_get_max_buffer_size();
+
+    /* In the situation where max_buffer_size is not four-byte aligned, reserve sufficient size for the buffer_size */
+    if (size > max_buffer_size - LIBSPDM_TEST_ALIGNMENT && (size & (alignment - 1)) != 0) {
```

Comment:
I dont understand why you mix the usage of `alignment` and `LIBSPDM_TEST_ALIGNMENT`.

If you already have alignment, then please always use it.

Reply:
Thanks, I have already modified it.